### PR TITLE
Enable different types of widgets on same horizontal axis

### PR DESCRIPTION
**Pull request by julienreliefapps**

### CODE REVIEW

**Comment by awalkowiak** [comment]
@jmiranda should this go to a different branch? (it's labeled as obnavtest2 on JIra)

File: src/js/actions/index.js [hunk 1]
```diff
@@ -276,19 +276,29 @@ function fetchNumberIndicator(
...
+      payload: {
+        id,
+        title: 'Loading...',
+        type: 'loadingNumber',
+        archived: indicatorConfig.archived,
+        enabled: indicatorConfig.enabled,
+      },
     });
   }
+  apiClient.get(url).then((res) => {
```
Comment (awalkowiak) [inline src/js/actions/index.js:290]: Previously this was triggered only if this was `indicatorConfig.enabled` true. Do you want to trigger `get` regardless of this?

**Review by awalkowiak — CHANGES_REQUESTED** [review]

File: src/js/actions/index.js (hunk 1 above)
Comment (julienreliefapps) [reply src/js/actions/index.js:290]: @awalkowiak oh no you are right, it's a mistake, I change that right now !

**Comment by awalkowiak** [comment]
@julienreliefapps @jmiranda should this target develop?

**Review by awalkowiak — APPROVED** [review]

**Comment by julienreliefapps** [comment]
@awalkowiak for the moment, I am not even sure that we have to merge it, because as the style will change a lot I have done that more to show how it would look. @jmiranda ?

**Comment by jmiranda** [comment]
@awalkowiak @julienreliefapps let's discuss in the ticket to include Kelsey. As far as I know it should go to the LLFB.

**Comment by awalkowiak** [comment]
@jmiranda I am taking this one to fix conflicts and to check if there is everything ok, after merging the other ticket with indicatior definitions

**Comment by awalkowiak** [comment]
@jmiranda as it is not required right now, I'm pulling this and closing this PR

**Pull request closed without merging**